src/pages/profile-page/profile-page.js

```diff
@@ -0,0 +1,312 @@
+    }
+  }, [userData]);
+
+  const { images } = user;
+
+  return (
+    <div>
+      <div className={classes.profile}>
+        <form className={classes.userForm}>
+          {userLoading ? (
+            <Loader />
+          ) : (
+            <>
+              <div className={classes.imageAndName}>
+                <div className={classes.imageContainer}>
+                  <img
+                    src={userImageUrl || ProfilePicture}
+                    alt='profile-logo'
+                    className={classes.userImage}
+                    onError={handleImageError}
+                  />
+                  <input
+                    type='file'
+                    className={classes.photoUpload}
+                    id='photoUpload'
+                    onChange={handleImageLoad}
+                    multiple={false}
+                    accept='image/*'
+                  />
+                  <label htmlFor='photoUpload' className={classes.uploadLabel}>
+                    <Button component='span' className={classes.uploadBtn}>
+                      {images
+                        ? images.thumbnail
+                          ? PROFILE_DATA[language].changePhoto
+                          : PROFILE_DATA[language].addPhoto
+                        : PROFILE_DATA[language].addPhoto}
+                    </Button>
+                  </label>
+                </div>
+                <div className={classes.userNames}>
+                  <TextField
+                    label={PROFILE_DATA[language].firstName}
+                    name='firstName'
+                    value={user.firstName || ''}
+                    className={classes.userInput}
+                    onChange={handleChange}
+                  />
+                  <TextField
+                    label={PROFILE_DATA[language].lastName}
+                    name='lastName'
+                    value={user.lastName || ''}
+                    className={classes.userInput}
+                    onChange={handleChange}
+                  />
+                </div>
+              </div>
+              <div className={classes.restOfUserInputs}>
+                <TextField
+                  label={PROFILE_DATA[language].email}
+                  name='email'
+                  value={user.email || ''}
+                  onChange={handleChange}
+                  className={classes.userInput}
+                />
+                <TextField
+                  label={PROFILE_DATA[language].phoneNumber}
+                  name='phoneNumber'
+                  value={user.phoneNumber ? user.phoneNumber : ''}
+                  onChange={handleChange}
+                  className={classes.userInput}
+                />
+                <TextField
+                  label={PROFILE_DATA[language].country}
+                  name='country'
+                  value={
+                    user.address
+                      ? user.address.country
```

Comment:
Why do you have `user.address.country` twice here?

Reply:
removed